accord-core/src/test/java/accord/impl/basic/PendingQueue.java

```diff
@@ -24,6 +24,11 @@ public interface PendingQueue
 {
     void add(Pending item);
     void add(Pending item, long delay, TimeUnit units);
+    default boolean remove(Pending item)
```

Comment:
This seems like an odd thing to have a default method that doesn't do the actual thing especially when it's always overridden.